mscore/plugin/pluginCreator.cpp

```diff
@@ -590,5 +601,27 @@ void PluginCreator::showManual()
             }
       manualDock->setVisible(!manualDock->isVisible());
       }
-}
+
+//---------------------------------------------------------
+//   focusInEvent
+//---------------------------------------------------------
+
+void PluginCreator::focusInEvent(QFocusEvent* event)
+      {
+      mscoreState = mscore->state();
+      mscore->changeState(STATE_DISABLED);
```

Comment:
Should not be `mscore->changeState(STATE_PLUGIN_CREATOR);` because the mainwindow should still get its actions disabled.